<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameConfigManager</class>
 <widget class="QWidget" name="GameConfigManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Btn_EditorNewJson">
       <property name="toolTip">
        <string>Make a new game configs container JSON file (a file contain multiple game configs) </string>
       </property>
       <property name="text">
        <string>New Games Config</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_EditorSaveJson">
       <property name="toolTip">
        <string>Save current editing game configs to JSON file</string>
       </property>
       <property name="text">
        <string>Save to Json</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_EditorOpenJson">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Open other game configs' JSON file.</string>
       </property>
       <property name="text">
        <string>Open Game Set Json</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string> Select a Game: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ComboBox_ConfigSelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_EditorNew">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_EditName">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Edit Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_EditorSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Save current editing to config selected in combo box.</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
